VK/VK/Controllers/AuthViewController.swift

```diff
@@ -0,0 +1,132 @@
+// AuthViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Экран авторизации
+final class AuthViewController: UIViewController {
+    private enum Constants {
+        static let segueName = "enterSegue"
+        static let alertTitleText = "Ошибка"
+        static let alertMessageText = "Введены неверные данные пользователя"
+        static let alertOkText = "OK"
+        static let login = "admin"
+        static let password = "12345"
+    }
+
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var scrollView: UIScrollView!
+    @IBOutlet private var loginTextField: UITextField!
+    @IBOutlet private var passwordTextField: UITextField!
+
+    // MARK: - LifeCycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        addObservers()
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        removeObservers()
+    }
+    
+    //MARK: - Navigation
+    override func shouldPerformSegue(withIdentifier identifier: String, sender: Any?) -> Bool {
+        if checkLogin() {
+            clearTextFields()
+            return true
+        } else {
+            clearTextFields()
+            showAlert(title: Constants.alertTitleText, message: Constants.alertMessageText, actionTitle: Constants.alertOkText)
+            return false
+        }
+    }
+
+    // MARK: - Private IBAction
```

Comment:
```suggestion
    // MARK: - Private Methods
```
марка IBAction -    только для IBAction